src/sections/Docker-Meshery/docker-extension-meshery.js

```diff
@@ -6,11 +6,17 @@ import Button from "../../reusecore/Button";
 import axios from "axios";
 
 import layer5_img from "../../assets/images/layer5/layer5-only/svg/layer5-white-no-trim.svg";
-const dockerDesktop = "../../assets/images/docker-extension/docker-extension-meshery.png";
+const mesheryworkdiagram =
+  "../../assets/images/mesherywork-diagram/mesherywork-diagram.svg";
+const dockercompose_logo =
+  "https://user-images.githubusercontent.com/7570704/166245883-b0660ace-1420-4150-8fd8-09eeaa8ff73b.png";
```

Comment:
This image also needs to be downloaded and then added to the images directory.

Reply:
Noted! At first, Direct URL of the Image was provided, and we're using StaticImage, It downloads and resize the Image at build time, Even if the Image is changed on the server, It will not be updated until we rebuild. But, It's done 👍✅

Reply:
That's the issue. Rebuild could be done by any change, and the image will get changed without getting noticed.
So, it is better to use images downloaded under the repo.